cockatrice/src/pictureloader.cpp

```diff
@@ -256,30 +325,33 @@ void PictureLoaderWorker::startNextPicDownload()
 
     cardBeingDownloaded = cardsToDownload.takeFirst();
 
-    QString picUrl = getPicUrl();
+    QString picUrl = cardBeingDownloaded.getCurrentUrl();
+
     if (picUrl.isEmpty()) {
         downloadRunning = false;
         picDownloadFailed();
     } else {
-        QUrl url(picUrl);
+        QUrl url(picUrl); // For now, just use the first one, like always.
```

Comment:
Can you elaborate on this comment? I don't get the meaning

Reply:
It's leftover from testing I had done previously.  I'll remove this.